react-interactive/src/ConfigurableTable/ConfigurableTable.tsx

```diff
@@ -258,7 +259,7 @@ export default function ConfigurableTable<T>(props: React.PropsWithChildren<IPro
                         disableAdd={false}
                     />
                 </Modal>
-                : (showSettings ? <Portal node={document && document.getElementById(props.SettingsPortal)}>
+                : (showSettings ? <Portal node={document != null && document.getElementById(props.SettingsPortal) != null}>
```

Comment:
```suggestion
                : (showSettings ? <Portal node={document?.getElementById(props.SettingsPortal) != null}>
```

Reply:
I realised this mistake after my last commit and rebased and force pushed before right before this comment.